src/fsdocs-tool/BuildCommand.fs

```diff
@@ -1406,10 +1404,13 @@ type CoreBuildOptions(watch) =
         // Adjust the user substitutions for 'watch' mode root
         let userRoot, userParameters =
             if watch then
-                let userRoot = sprintf "http://localhost:%d/" this.port_option
+                let userRoot =
```

Comment:
Would

```fsharp
let userRoot =
    match userParametersDict.TryGetValue(ParamKeys.root) with
    | true, root -> root
    | false, _ -> sprintf "http://localhost:%d/" this.port_option
```

work for you here?
I believe it can be an easy way to make 
```
fsdocs watch --parameters root "/"                                                                                                                                                                               
```

work and would be the least disruptive for now.

We don't feel good enough about the new setting.

Reply:
I will test and let you know! Thanks for talking it through!

Reply:
Sure thing, thanks for your patience!